legal-api/src/legal_api/services/filings/validations/validation.py

```diff
@@ -88,6 +89,18 @@ def validate(business: Business, filing_json: Dict) -> Error:  # pylint: disable
                                                       'path': '/filing/specialResolution'}])
         if err:
             return err
+    elif 'specialResolution' in filing_json['filing'].keys() and legal_type in [Business.LegalTypes.COOP.value]:
+        err = special_resolution_validate(business, filing_json)
+        if err:
+            return err
+
+        if 'changeOfName' in filing_json['filing'].keys():
+            err = con_validate(business, filing_json)
+        if 'alteration' in filing_json['filing'].keys():
+            err = alteration_validate(business, filing_json)
+
+        if err:
+            return err
```

Comment:
Why do you need this elif? below else loop through these validation anyway

Reply:
I missed one logic here, just added now. If it is a SR for COOP then we either need CoN or Alteration along with it.